techbuzz/src/main/java/com/sivalabs/techbuzz/users/domain/services/UserService.java

```diff
@@ -66,4 +72,12 @@ public void verifyEmail(String email, String token) {
         user.setVerificationToken(null);
         userRepository.updateVerificationStatus(user);
     }
+
+    @CacheEvict(cacheNames = "user", allEntries = true)
+    public Optional<UserDTO> getUserDTO(String emailID) {
```

Comment:
This can be simplified to:

```
public Optional<UserDTO> getUserDTO(String emailID) {
    return this.getUserByEmail(emailID).map(userDTOMapper::toDTO);
}
```

Reply:
yes thanks for simpler suggestion done